web/packages/teleport/src/Login/Login.tsx

```diff
@@ -78,3 +90,114 @@ export function LoginComponent({
     </>
   );
 }
+
+const LicenseBox = styled(Box)`
+  width: 100%;
+  max-width: 550px;
+  padding: ${props => props.theme.space[4]}px;
+  background-color: ${props => props.theme.colors.levels.surface};
+  border-radius: ${props => props.theme.radii[3]}px;
+  margin: auto;
+`;
+
+function LicenseAcknowledgement({
```

Comment:
If I log out, I'm taken back to this license acknowledgement (it seems the local storage value gets removed).

I would not expect this. Once I've acknowledged it once, I shouldn't see it again.

Reply:
Updated